lib/contiguous.cpp

```diff
@@ -0,0 +1,52 @@
+#include "flag_gems/operators.h"
+#include "flag_gems/utils.h"
+
+#include <iostream>
+#include "c10/cuda/CUDAStream.h"
+#include "triton_jit/triton_jit_function.h"
+
+namespace flag_gems {
+using namespace triton_jit;
+
+at::Tensor contiguous(const at::Tensor &self, at::MemoryFormat memory_format) {
```

Comment:
Since our current implementation does not support ChannelLast and ChannelLast3d format, it is necessary to raise an error if the memory format passed in is not supported. We can leave it for the future.